<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Slapsh</class>
 <widget class="QMainWindow" name="Slapsh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loan</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius:14px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(255, 255, 255);
border-radius:10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>711</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Welcome</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>310</y>
         <width>671</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar
{
border: solid grey;
border-radius: 10px;
color: black;
   background-color: rgb(255, 255, 255);
}
QProgressBar::chunk 
{
background-color: #ff0000;
border-radius :10px;
}   </string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>360</y>
         <width>661</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Starting.....</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>711</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Loan Mangement Application</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
